### PR TITLE
Adding codebook env and cicd

### PR DESCRIPTION
**Pull request by yambottle**

### CODE REVIEW

**Review by guzman-raphael — CHANGES_REQUESTED** [review]
@yambottle Great work! :clap: 

Would you just double check on the cloning point?

**Comment by kabilar** [comment]
Thank you, @yambottle!  I will defer to @guzman-raphael to review this work.

Prior to merging, let's decide on the discussion from [pull request #56](https://github.com/datajoint/workflow-array-ephys/pull/56#issuecomment-1114093572) in workflow-array-ephys.  Thanks!

File: docker/docker-compose-codebook_env.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      context: .
+      dockerfile: codebook.Dockerfile
+      args:
+        - JHUB_VER
+        - PY_VER
+        - DIST
+        - DEPLOY_KEY
+        - REPO_OWNER
+        - REPO_NAME
+    image: registry.vathes.com/${REPO_OWNER}/codebook-${REPO_NAME}:jhub${JHUB_VER}-py${PY_VER}-${DIST}-v${WORKFLOW_VERSION}
```
Comment (kabilar) [inline docker/docker-compose-codebook_env.yaml:15]: After discussion with @yambottle @guzman-raphael, we will publish to open-source DataJoint DockerHub.

**Comment by yambottle** [comment]
> Hi @yambottle - Are there still things to address here? I'm interested in getting some of these pieces implemented across other element workflows. Lmk if I can support that process

It needs a little bit change since I updated the SciOps CICD, we need to follow the same template.

**Comment by kabilar** [comment]
Hi @yambottle, since you are now working on centralizing the CI for the Workflows and Elements, would these updates still be applicable?  If not, we can close this pull request.  Thanks!

**Pull request closed without merging**